test/mpi/pt2pt/testlist.def

```diff
@@ -50,3 +50,11 @@ dtype_send 2
 recv_any 2
 irecv_any 2
 large_tag 2
+
+# Heterogeneous memory tests
+sendflood 8 env=MPIR_CVAR_MEMBIND_NUMA_ENABLE="YES" env=MPIR_CVAR_MEMBIND_TYPE_LIST="FASTBOXES:AUTO" env=MPIR_CVAR_MEMBIND_POLICY_LIST="FASTBOXES:BIND" env=MPIR_CVAR_MEMBIND_FLAGS_LIST="FASTBOXES:STRICT" timeLimit=600
```

Comment:
It looks like these additional quotes are causing the XML parser to barf. Can you remove them? E.g. ```sendflood 8 env=MPIR_CVAR_MEMBIND_NUMA_ENABLE=YES env=MPIR_CVAR_MEMBIND_TYPE_LIST=FASTBOXES:AUTO env=MPIR_CVAR_MEMBIND_POLICY_LIST=FASTBOXES:BIND env=MPIR_CVAR_MEMBIND_FLAGS_LIST=FASTBOXES:STRICT timeLimit=600```